src/services/textChanges.ts

```diff
@@ -72,6 +72,11 @@ namespace ts.textChanges {
      */
     export type ConfigurableStartEnd = ConfigurableStart & ConfigurableEnd;
 
+    export const useNonAdjustedPositions: ConfigurableStartEnd = {

```

Comment:
No longer needs to be exported

Reply:
I left it exported because I'll be consuming it in another PR.